### PR TITLE
Fix links, add materialized views

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: _posts/2021-01-08-2020-review.md [hunk 1]
```diff
@@ -194,6 +194,7 @@ There is so much more to capture, and you really would have to read all the
 [release notes](/docs/current/release.html#releases-2020) in detail to know it
 all. To safe you from that, here are a few more noteworthy changes:
 
+* Support for materialized views
```
Comment (findepi) [inline _posts/2021-01-08-2020-review.md:197]: This could be in SQL section. but @martint would know better

Comment (mosabua) [reply _posts/2021-01-08-2020-review.md:197]: Yeah .. I wasnt sure either so for now I added it that everything else section...

**Review by findepi — APPROVED** [review]
LGTM % placement (https://github.com/trinodb/trino.io/pull/151/files#r555887119)

@martint PTAL

File: _posts/2021-01-08-2020-review.md (hunk 1 above)
Comment (martint) [reply _posts/2021-01-08-2020-review.md:197]: Currently, they are very iceberg-specific, so I would phrase as:

"Experimental support for materialized views in Iceberg connector".

Comment (mosabua) [reply _posts/2021-01-08-2020-review.md:197]: Ok

Comment (mosabua) [reply _posts/2021-01-08-2020-review.md]: Done

**Comment by mosabua** [comment]
@martint can you merge this please?

**Review by findepi — APPROVED** [review]

**Comment by findepi** [comment]
Merged, thanks!

**Pull request merged**